src/neptune/utils.py

```diff
@@ -14,8 +14,9 @@
 # limitations under the License.
 #
 """Utility functions to support ML metadata logging with neptune.ai."""
-__all__ = ["stringify_unsupported", "stop_synchronization_callback"]
+__all__ = ["stringify_unsupported", "stop_synchronization_callback", "is_unsupported_float"]
```

Comment:
That's a module for user-exposed utils. This function is much more internal.

Reply:
True, deleted from here 😄

Reply:
Looks like it's still here :D.

Reply:
Moved back because of the comment you left later on, but decided to move to internal common module 😉

Reply:
Sorry for confusing you, one thing is the proper placing another is to remember about `__all__` filling up.